health-connect/HealthConnectSample/app/src/main/java/com/example/healthconnectsample/presentation/navigation/Screen.kt

```diff
@@ -36,5 +38,6 @@ enum class Screen(val route: String, val titleId: Int, val hasMenuItem: Boolean
     InputReadings("input_readings", R.string.input_readings),
     DifferentialChanges("differential_changes", R.string.differential_changes),
     PrivacyPolicy("privacy_policy", R.string.privacy_policy, false),
-    SettingsScreen("settings_screen", R.string.settings)
+    SettingsScreen("settings_screen", R.string.settings),
+    RecordListScreen("record_list", R.string.record_list),
```

Comment:
add `false` so it doesn't show up in the left hand menu (this was causing the crash)

Reply:
Good catch - done